backend/src/services/aws/lambda/lambda.service.ts

```diff
@@ -93,6 +100,28 @@ class Lambda {
       }),
     );
   }
+
+  public async runFunction(name: string): Promise<string> {
+    const { Payload, StatusCode } = await this.#lambdaClient.send(
+      new InvokeCommand({
+        FunctionName: name,
+      }),
+    );
+    const hasPayload = Boolean(Payload);
+
+    if (StatusCode !== HttpCode.OK || !hasPayload) {
+      throw new SLCError({
+        status: HttpCode.INTERNAL_SERVER_ERROR,
+        message: ExceptionMessage.FUNCTION_NOT_RUN,
+      });
+    }
```

Comment:
Two things here that are not quite correct me:
1. INTERNAL_SERVER_ERROR as a status. In 99% of cases there is no need to **explicitly** throw 500 error. 500 code is ok for an unhandled exceptions. But here you have explicitly defined catch
2. message is obscure. There are a lof of reasons for function to fail on run. Imagine you are using lambda service as a utility tool without knowing how it works internally. And than you receive FUNCTION_NOT_RUN exception. How would you debug it?)

Additionally, it seems like status code does not represent error. For example, everyone from [this thread](https://github.com/aws/aws-sdk-js-v3/issues/2007) use try/catch.
If you still want to provide more detailed exception you can use try/catch in s3.service createBucket method as an example